courseworks/week4/ZhigaoXIE/Q1.txt

```diff
@@ -0,0 +1,6 @@
+using LinearAlgebra
+
+a = 5.0
+b = 3.0
+
+cond(a/b) = 1.0
```

Comment:
Using specific numbers is a good way to check your result, but we should use the definition of condition numbers as shown below.
<img width="1097" alt="image" src="https://user-images.githubusercontent.com/45324209/226504743-fdb90e33-4633-4cdb-9af0-06b5af96b0a7.png">